src/word_analysis.py

```diff
@@ -0,0 +1,22 @@
+from wordcloud import WordCloud
+import numpy as np
+from textblob import TextBlob
+import matplotlib.pyplot as plt
+
+
+def generate_word_cloud(df, column):
```

Comment:
I don't think you need these methods as scripts in here
They are stuff for your research, that show what are the characteristics of your dataset and stuff